solr/core/src/java/org/apache/solr/cloud/api/collections/SplitShardCmd.java

```diff
@@ -199,8 +200,17 @@ public boolean split(ClusterState clusterState, ZkNodeProps message, NamedList<O
             "SplitShardCmd: verify that there is enough space on disk to create sub-shards for slice: {}",
             parentShardLeader);
         t = timings.sub("checkDiskSpace");
-        checkDiskSpace(
-            collectionName, slice.get(), parentShardLeader, splitMethod, ccc.getSolrCloudManager());
+        boolean skipFreeSpaceCheck = message.getBool(SKIP_FREE_SPACE_CHECK, false);
```

Comment:
See the System.getProperty thing above?  Let's just combine this into one so that the system property is the default.
CC @heythm

Reply:
Oh, I haven't seen the other check, I've just applied my earlier changes. Now I wonder if this per-call granularity is necessary in this logic. I can see the advantage that retrying a shard split with this error does not require a restart with this parameter.